.buildkite/pipeline.yaml

```diff
@@ -11,21 +11,22 @@ steps:
           - "automatedtests/buildout/outputs/apk/androidTest/**/*.apk"
           - "automatedtests/buildout/outputs/apk/*/debug/automatedtests-*-debug.apk"
 - wait
-- agents: [dind=true,queue=beta]
-  command: docker run --rm -v $(pwd):/data -e TEST_APK=/data/automatedtests/buildout/outputs/apk/androidTest/r2_9_6/debug/automatedtests-r2_9_6-debug-androidTest.apk -e APP_APK=/data/automatedtests/buildout/outputs/apk/r2_9_6/debug/automatedtests-r2_9_6-debug.apk -e SAUCE_ACCESS_KEY=$SAUCE_ACCESS_KEY muxinc/saucelabs-virtual-test-runner:0.1.2
-  label: Test Exoplayer r2_9_6 on Android GoogleAPI Emulator, platform-version 10.0
+- agents: [ dind=true,queue=beta ]
```

Comment:
We generally don't include whitespace on the interior of the YAML brackets, can you remove that in the place where it was added?